kotlinx-coroutines-core/common/src/flow/operators/Delay.kt

```diff
@@ -71,6 +73,34 @@ public fun <T> Flow<T>.debounce(timeoutMillis: Long): Flow<T> {
     }
 }
 
+/**
+ * Returns a flow that mirrors the original flow, but filters out values
+ * that are followed by the newer values within the given [timeout].
+ * The latest value is always emitted.
+ *
+ * Example:
+ * ```
+ * flow {
+ *     emit(1)
+ *     delay(90.0.milliseconds)
```

Comment:
why not to use extensions on `Int` instead of `Double`, like `90.milliseconds` instead of `90.0.milliseconds` ?